dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -393,7 +393,7 @@ class CompileTest {
         watch.start();
         await flutter('build', options: options);
         watch.stop();
-        final String appPath =  '$cwd/build/ios/Release-iphoneos/Runner.app/';
+        final String appPath =  '$cwd/build/ios/Release-iphoneos/sample_flutter_app.app/';
```

Comment:
This is the line that was changed from https://github.com/flutter/flutter/pull/46838.
`dart bin/run.dart -t basic_material_app_ios__compile` passes on my machine now.